### PR TITLE
Perth TCU

### PR DESCRIPTION
**Pull request by tylerthetiletiler**
Perth TCU Finalisation

Perth Tower Amendments

### CODE REVIEW

**Review by chrisg06 — DISMISSED** [review]

**Comment by chrisg06** [comment]
approved

**Review by zkgell — DISMISSED** [review]
approved

File: docs/terminal/perth.md [hunk 1]
```diff
@@ -4,14 +4,55 @@
...
+| **Perth Approach** |**PHA**| **Perth Approach**  | **123.600** | **PH_APP**| 
+| Perth Departures†  |PHD| Perth Departures  | 118.700 | PH_DEP |
+| Perth Flow† | PHF |   |    | PH-FLW_CTR  |
+
+† **Non-standard position** – may only be used in accordance with [VATPAC Ratings and Controller Positions Policy](https://cdn.vatpac.org/documents/policy/Controller+Positions+and+Ratings+Policy+v5.2.pdf){target=new}
+
+## Airspace
+The Perth TMA spans a 36nm Radius around PH from SFC-FL245.   
+PH TCU is responsible for the Perth TMA, except:    
+a) Class G cut out to the south
```
Comment (alexd-332) [inline docs/terminal/perth.md:18]: PH TCU is responsible for the Class G cutout

File: docs/terminal/perth.md [hunk 2]
```diff
@@ -23,4 +64,97 @@ The divisions of the airspace between **PHA**, and **PHD** change based on the R
...
   <figcaption>21/24 TCU Structure</figcaption>
-</figure>
\ No newline at end of file
+</figure>
+
+## Coordination
+
+### PH TCU / ENR
+#### Departures
+Voiceless coordination is in place from AD TCU to PIY (and subsectors) for aircraft:
```
Comment (alexd-332) [inline docs/terminal/perth.md:73]: and assigned a SID, or maybe "tracking via SID terminus"

**Review by alexd-332 — CHANGES_REQUESTED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by zkgell — APPROVED** [review]
approved

**Review by zachbf — APPROVED** [review]

**Pull request merged**